nati_x86_64_defconfig: enable switchtec

wchianhong opened the pull request:
Enabling CONFIG_NTB_SWITCHTEC module to natively support Microchip switch without having to install switchtec-kernel separately. NI is planning on using Microchip PCIe switch
By extension, Linux RT will also require to have CONFIG_NTB enabled to support the Microchip PCIe switches

wchianhong commented:
@gratian Can you please help to review this change? FYI @nikhalim

chaitu236 commented on arch/x86/configs/nati_x86_64_defconfig:
```diff
@@ -625,3 +625,5 @@ CONFIG_TRACER_SNAPSHOT_PER_CPU_SWAP=y
 # CONFIG_EARLY_PRINTK is not set
 CONFIG_IO_DELAY_0XED=y
 CONFIG_UNWINDER_GUESS=y
+CONFIG_NTB=y
```
For changing defconfig, the procedure is to generate `.config` (by running `make nati_x86_64_defconfig`), using `make menuconfig` to enable the new config options, `make savedefconfig` and replace `nati_x86_64_defconfig` with the generated file. Was this process followed for these changes?

wchianhong replied:
With my latest push, i have followed the process that you have specified

chaitu236 reviewed (changes requested):
Please fix the commit message as per kernel standards. You can refer to [this](https://github.com/ni/linux/pull/196#issuecomment-2666504102) comment from another PR.

In the PR description, please mention how this was tested. If there is an NI work item that prompted this change, please also link it (e.g., refer [this PR](https://github.com/ni/linux/pull/174)).

Also, I see that you're part of NI GitHub organization, can you get your GitHub ID added [here](https://dev.azure.com/ni/DevCentral/_git/AppCentral.wiki?version=GC2ef30474e33cdfb85b32696ddebf716327b8ca2c&path=/Tools/GitHub/NI-GitHub-IDs.md&_a=preview) so we know how to contact you internally.

wchianhong commented:
Related WI: https://dev.azure.com/ni/DevCentral/_workitems/edit/3029794

I have also added my GitHub ID to the NI GitHub organization

**Testing**

- [ /] Tested on a PXIe-8842 controller with built kernel
- [ /] Tested that Microchip switches on MAUI chassis connected to the 8842 are able to be detected in Linux RT 
        `ls /dev/switchtec*`

wchianhong committed:
nati_x86_64_defconfig: enable switchtec

Enabling CONFIG_NTB_SWITCHTEC module to natively support
Microchip switch without manually installing switchtec-kernel.
NI is planning on using Microchip PCIe switch.
By extension, Linux RT will also require to have CONFIG_NTB
enabled to support the Microchip PCIe switches.

Signed-off-by: Brian Wong <brian.wong@emerson.com>

wchianhong commented:
@chaitu236 I have made amendments based upon your comments from last week. I hope that they are acceptable. Please help to take another look, thanks. FYI for @gratian and @nikhalim
